.github/actions/verify-token/src/predicate.ts

```diff
@@ -138,12 +137,25 @@ export function createPredicate(
   };
 
   const payload = github.context;
+  // NOTE: see example at https://github.com/slsa-framework/slsa/blob/main/docs/github-actions-workflow/examples/v0.1/example.json.
   const predicate: SLSAv1Predicate = {
     buildDefinition: {
       buildType: DELEGATOR_BUILD_TYPE,
       externalParameters: {
+        // Inputs to the TRW, which define the interface of the builder for the
```

Comment:
+1 I'm in agreement that because this is provenance for a tool action invocation, that the "top-level inputs" we consider here are the TRW inputs. It took me a while to reconcile that (because technically they are "derived", but I think we should take the stance that the BUILD is for the TRW invocation, not for a higher level workflow invocation)